lib/internal/bootstrap/switches/is_main_thread.js

```diff
@@ -6,8 +6,8 @@ const rawMethods = internalBinding('process_methods');
 // TODO(joyeecheung): deprecate and remove these underscore methods
 process._debugProcess = rawMethods._debugProcess;
 process._debugEnd = rawMethods._debugEnd;
-process._startProfilerIdleNotifier = rawMethods._startProfilerIdleNotifier;
-process._stopProfilerIdleNotifier = rawMethods._stopProfilerIdleNotifier;
+process._startProfilerIdleNotifier = () => {};
```

Comment:
It would be good to include a comment here about why these are non-ops, or at least a link to the github conversation for those who come along later

Reply:
Done.